src/main/java/edu/kit/kastel/sdq/artemis4j/grading/Assessment.java

```diff
@@ -47,13 +54,29 @@ public class Assessment extends ArtemisConnectionHolder {
     private static final FormatString GLOBAL_FEEDBACK_MISTAKE_TYPE_HEADER_NONSCORING =
             new FormatString(new MessageFormat("    * {0}:"));
     private static final FormatString GLOBAL_FEEDBACK_ANNOTATION =
-            new FormatString(new MessageFormat("        * {0} " + "at line {1,number}"));
+            new FormatString(new MessageFormat("        * {0} at line {1}"));
+    private static final FormatString GLOBAL_FEEDBACK_ANNOTATION_MULTIPLE =
+            new FormatString(new MessageFormat("        * {0} at lines {1}"));
     private static final FormatString GLOBAL_FEEDBACK_ANNOTATION_CUSTOM_PENALTY =
-            new FormatString(new MessageFormat("        * {0} at line {1,number} ({2,number,##.###}P)"));
+            new FormatString(new MessageFormat("        * {0} at line {1} ({2,number,##.###}P)"));
+    private static final FormatString GLOBAL_FEEDBACK_ANNOTATION_CUSTOM_PENALTY_MULTIPLE =
+            new FormatString(new MessageFormat("        * {0} at lines {1} ({2,number,##.###}P)"));
     private static final FormatString GLOBAL_FEEDBACK_LIMIT_OVERRUN = new FormatString(
             new MessageFormat("    * Note:" + " The sum of penalties hit the limits for this rating group."));
     private static final FormatString NO_FEEDBACK_DUMMY = new FormatString("The tutor has made no annotations.");
 
+    /**
+     * The global feedback containing a list of all annotations that were made, might be too large. In this case,
+     * the feedback is split into multiple parts. Artemis will display feedbacks that subtract points in a different
+     * section than feedbacks that do not subtract points.
+     * <p>
+     * With only the first feedback deducting points, the rest would be displayed in the info section. To prevent this,
+     * the other parts are passed a negative score that is as close to zero as possible.
+     * <p>
+     * This negative score is defined by the following constant.
+     */
+    private static final double GLOBAL_FEEDBACK_OTHER_PARTS_SCORE = -Double.MIN_VALUE;
```

Comment:
Nice hack :D Could you please check whether negative zero (https://en.wikipedia.org/wiki/Signed_zero) would work as well? If not, could you add one line to the docs that explains that :)

Reply:
I tested that a few weeks ago, it doesn't work.